src/lib/layout/Searchbar.tsx

```diff
@@ -85,16 +89,27 @@ const Searchbar = () => {
   const [results, setResults] = useState<SearchResultType[]>([]);
 
   const boxRef = useRef<HTMLDivElement>(null);
+  const chainConfig = getChainConfig();
+
+  let placeholder = "Search by Wallet Address / Contract Address / Tx Hash / ";
+  placeholder += chainConfig.isWasm ? "Code ID" : "Block";
```

Comment:
no block search for wasm?

Reply:
Currently no block search for wasm supported chain. Will add block search later